http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala

```diff
@@ -117,15 +147,27 @@ object NonLeoMessageSubscriber {
     Decoder.forProduct2("clusterId", "project")(DeleteKubernetesClusterMessage.apply)
 
   implicit val googleLabelsDecoder: Decoder[GoogleLabels] =
-    Decoder.forProduct3("instance_id", "project_id", "zone")(GoogleLabels.apply)
+    Decoder.forProduct2("instance_id", "zone")(GoogleLabels.apply)
   implicit val googleResourceDecoder: Decoder[GoogleResource] = Decoder.forProduct1("labels")(GoogleResource.apply)
-  Decoder.forProduct2("textPayload", "resource")(CryptoMining.apply)
-  implicit val cryptoMiningDecoder: Decoder[CryptoMining] =
-    Decoder.forProduct2("textPayload", "resource")(CryptoMining.apply)
+  implicit val cryptoMiningDecoder: Decoder[CryptoMining] = Decoder.instance { c =>
+    for {
+      textpayload <- c.downField("textPayload").as[String]
+      resource <- c.downField("resource").as[GoogleResource]
+      logName <- c.downField("logName").as[String]
+      splitted = logName.split("/")
```

Comment:
What is the `logName` format?

Reply:
looks like `  "logName": "projects/general-dev-billing-account/logs/cryptomining",`, I'll add a comment